index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="main.css">
+	<title>Tic Tac Toe</title>
+</head>
+<body>
+	<div id="layer">
+	
+	<p>Tic-Tac-Toe</p>
+
+	<div id="gameboard">	
+		<div class='tile' id="one"></div>
+		<div class='tile' id="two"></div>
+		<div class='tile' id="three"></div>
+		<br/>
+		<div class='tile' id="four"></div>
+		<div class='tile' id="five"></div>
+		<div class='tile' id="six"></div>
+		<br/>
+		<div class='tile' id="seven"></div>
+		<div class='tile' id="eight"></div>
+		<div class='tile' id="nine"></div>
+	</div>
+
+<button id="button">Reset</button>
+
+</div>
+ </body>
+<script type="text/javascript" src="script.js"></script>
+</html>
```

Comment:
Excellent layout with your HTML. Not a huge deal, but generally `<br>` tags are reserved for inside of text elements (`<p>` or header tags), but as you saw, they also work for making items display as block.